packages/core/content-manager/server/src/controllers/relations.ts

```diff
@@ -93,7 +93,7 @@ const validateLocale = (sourceUid: UID.Schema, targetUid: UID.ContentType, local
 
   let validatedLocale = locale;
 
-  if (!targetModel || !isTargetLocalized) validatedLocale = undefined;
+  if (!targetModel && !isTargetLocalized) validatedLocale = undefined;
```

Comment:
I think we can safely remove this line... I tried it locally and it was working.

We only use locale to filter if `isTargetLocalized` or `isSourceLocalized` is true. (depending on the case) So this looks like not necessary

Reply:
agreed